diff -Naur cnr.orig/cinelerra/main.C cnr/cinelerra/main.C
--- cnr.orig/cinelerra/main.C	2011-04-24 18:42:17.000000000 +0400
+++ cnr/cinelerra/main.C	2011-04-24 23:50:10.377492251 +0400
@@ -41,7 +41,7 @@
 #include <stdlib.h>
 #include <string.h>
 
-#include <locale.h>
+#include <langinfo.h>
 
 #define PACKAGE "cinelerra"
 #define LOCALEDIR "/usr/share/locale"
@@ -81,32 +81,36 @@
 // detect an UTF-8 locale and try to use a non-Unicode locale instead
 // <---Beginning of dirty hack
 // This hack will be removed as soon as Cinelerra is UTF-8 compliant
-    char *s, *language;
+    char *s, *language, *mess;
 
 // Query user locale
-    if ((s = getenv("LC_ALL"))  || 
+	if ((s = getenv("LANG"))  || 
+		(s = getenv("LC_ALL")) || 
 		(s = getenv("LC_MESSAGES")) || 
-		(s = getenv("LC_CTYPE")) || 
-		(s = getenv ("LANG")))
-    {
+		(s = getenv("LC_CTYPE")))
+	{
 // Test if user locale is set to Unicode        
-        if (strstr(s, ".UTF-8"))
-        {
+		if (strstr(s, "ru_RU.UTF-8"))
+		{
+			mess = getenv("LC_MESSAGES");
+			setlocale (LC_ALL, "ru_RU.koi8-r");
+			if (mess)
+				setlocale (LC_MESSAGES, mess);
+		}
+		else if (strstr(s, ".UTF-8"))
+		{
 // extract language  from language-charset@variant
-          language = strtok (s, ".@");
+			language = strtok (s, ".@");
 // set language as the default locale
-          setenv("LANG", language, 1);
-        }
-    }
+			setenv("LANG", language, 1);
+			s = getenv ("LANG");
+			setlocale (LC_ALL, s);
+		}
+	}
 // End of dirty hack --->
-
-
-
-
-	bindtextdomain (PACKAGE, LOCALEDIR);
-	textdomain (PACKAGE);
-	setlocale (LC_MESSAGES, "");
-	setlocale (LC_CTYPE, "");
+ 
+	bind_textdomain_codeset(PACKAGE,nl_langinfo(CODESET));
+	textdomain(PACKAGE);
 
 	for(int i = 1; i < argc; i++)
 	{
diff -Naur cnr.orig/guicast/bcresources.C cnr/guicast/bcresources.C
--- cnr.orig/guicast/bcresources.C	2011-04-24 22:35:16.000000000 +0400
+++ cnr/guicast/bcresources.C	2011-04-25 11:55:54.766631315 +0400
@@ -61,23 +61,23 @@
 	new VFrame(file_column_png)
 };
 
-char* BC_Resources::small_font = N_("-*-helvetica-medium-r-normal-*-10-*");
-char* BC_Resources::small_font2 = N_("-*-helvetica-medium-r-normal-*-11-*");
-char* BC_Resources::medium_font = N_("-*-helvetica-bold-r-normal-*-14-*");
-char* BC_Resources::medium_font2 = N_("-*-helvetica-bold-r-normal-*-14-*");
-char* BC_Resources::large_font = N_("-*-helvetica-bold-r-normal-*-18-*");
-char* BC_Resources::large_font2 = N_("-*-helvetica-bold-r-normal-*-20-*");
-
-char* BC_Resources::small_fontset = "6x12,*";
-char* BC_Resources::medium_fontset = "7x14,*";
-char* BC_Resources::large_fontset = "8x16,*";
+char* BC_Resources::small_font = N_("-*-fixed-medium-r-normal-*-10-*");
+char* BC_Resources::small_font2 = N_("-*-*-medium-r-normal-*-10-*");
+char* BC_Resources::medium_font = N_("-*-fixed-medium-r-normal-*-14-*");
+char* BC_Resources::medium_font2 = N_("-*-*-medium-r-normal-*-14-*");
+char* BC_Resources::large_font = N_("-*-fixed-medium-r-normal-*-20-*");
+char* BC_Resources::large_font2 = N_("-*-*-medium-r-normal-*-20-*");
+
+char* BC_Resources::small_fontset = "-*-*-medium-r-normal-*-10-*,*";
+char* BC_Resources::medium_fontset = "-*-*-medium-r-normal-*-14-*,*";
+char* BC_Resources::large_fontset = "-*-*-medium-r-normal-*-20-*,*";
 
 char* BC_Resources::small_font_xft = N_("-*-luxi sans-*-r-*-*-12-*-*-*-*-*-*-*");
-char* BC_Resources::small_font_xft2 = N_("-microsoft-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
+char* BC_Resources::small_font_xft2 = N_("-*-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
 char* BC_Resources::medium_font_xft = N_("-*-luxi sans-*-r-*-*-16-*-*-*-*-*-*-*");
-char* BC_Resources::medium_font_xft2 = N_("-microsoft-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
+char* BC_Resources::medium_font_xft2 = N_("-*-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
 char* BC_Resources::large_font_xft = N_("-*-luxi sans-bold-r-*-*-20-*-*-*-*-*-*-*");
-char* BC_Resources::large_font_xft2 = N_("-microsoft-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
+char* BC_Resources::large_font_xft2 = N_("-*-verdana-*-*-*-*-*-*-*-*-*-*-*-*");
 
 suffix_to_type_t BC_Resources::suffix_to_type[] = 
 {
@@ -526,7 +526,7 @@
 
 	audiovideo_color = RED;
 
-	use_fontset = 0;
+	use_fontset = 1;
 
 // Xft has priority over font set
 #ifdef HAVE_XFT
